src/libcore/num/mod.rs

```diff
@@ -2673,12 +2664,12 @@ mod ptr_try_from_impls {
     try_from_both_bounded!(isize, i8, i16);
     try_from_unbounded!(isize, i32, i64, i128);
 
-    rev!(try_from_unbounded, usize, u8, u16, u32);
+    rev!(try_from_unbounded, usize, u16, u32);
     rev!(try_from_upper_bounded, usize, u64, u128);
     rev!(try_from_lower_bounded, usize, i8, i16, i32);
     rev!(try_from_both_bounded, usize, i64, i128);
 
-    rev!(try_from_unbounded, isize, u8, u16);
```

Comment:
This impl needs to remain, it's the `i8` to `isize` impl below that needs to be removed. The same two impls need to be removed for the 16-bit case as well. Basically ba74a8665d2b7771b6fd0499dc050c1bf075fca9 needs to be reverted.